Arch/README.md

```diff
@@ -0,0 +1,15 @@
+# Pros
+* **Customization capability**
+      After loading the ISO file for installation, the user can freely monitor and control various functional aspects, which is a fundamental distinguishing feature of Arch Linux.
+* **Flexible package management and integrity**
+      LINUX ARCH supports PACMAN management. This technology provides the ability to upgrade the entire system with a single pacman command - Syu. Loading of new versions of packages is performed by one click of pacman - S package-name.
+* **Arch Wiki**
+      your source for Arch Linux documentation on the web.
```

Comment:
Most distros have a wiki, so this alone isn't a pro. The real pro for Arch is that its wiki is rich, detailed, and up-to-date, making it superior to any other distro's wiki . Maybe we can say something about that here!